README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
Have an idea about laws. We can implement `chainRec` using `chain`, right? Except that implementation will not be stack safe. But in ideal world they must be identical, at leas that what we want from `chainRec`. So we could say exactly that in a law. It could look like this:

> ```js
> T.chainRec(f, i)
> ```
> is equivalent to
>
> ```js
> const g = x => {
>   const r = f(next => ({next}), done => ({done}), x);
>   return 'next' in r ? g(r.next) : T.of(r.done)
> }
> T.of(i).chain(g)
> ```

Not sure that I implemented it using `chain` correctly, but you get the idea.